src/main/java/core/basesyntax/service/implementation/IncreaseOperationImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.dto.FruitRecordDto;
+import core.basesyntax.model.Storage;
+import core.basesyntax.service.FruitOperationHandler;
+
+public class IncreaseOperationImpl implements FruitOperationHandler {
+
+    @Override
+    public int apply(FruitRecordDto fruitRecordDto) {
+        Integer currentQuantity = Storage.getFruits().get(fruitRecordDto.getFruitName());
+        if (fruitRecordDto.getQuantity() > 0) {
```

Comment:
Why do you use this check (fruitRecordDto.getQuantity() > 0)  here? you have already checked the validity of data in FruitParserImpl class.